code/game/gamemodes/game_mode.dm

```diff
@@ -10,6 +10,7 @@ var/global/list/additional_antag_types = list()
 	var/probability = 0
 
 	var/required_players = 0                 // Minimum players for round to start if voted in.
+	var/required_players_max = 0			 // Maximum players for round to start for secret voting. 0 means "doesn't matter"

```

Comment:
That variable name is not intuitive
Just name it max_players

Reply:
Variable name suggests to me that "There can be no more than x players", which might be problematic in the future.